lib/prompts/input.js

```diff
@@ -82,7 +83,7 @@ Prompt.prototype.onEnd = function( state ) {
     this.status = "answered";
 
     // Re-render prompt
-    this.clean(1).render();
+    this.clean( this.lines() ).render();
```

Comment:
@SBoudrias Do you have any insight on how to get this function (`Prompt.prototype.onEnd`) called in the unit tests? Currently only `onKeypress` seems to be called there, even when emitting a line event.